src/components/forms/inputYear/InputYear.tsx

```diff
@@ -0,0 +1,62 @@
+import styles from '../../../calculator/calculator.module.css';
```

Comment:
It reduces prop drilling because we omit one component like InputYear and make certain all logic related to the form is in one place. 💡 

<img width="539" alt="Screenshot 2024-08-19 at 10 57 15" src="https://github.com/user-attachments/assets/c97b37c4-b73e-4f4d-a5b0-2e3ea40bc661">

Reply:
That makes sense :+1: So with this model you would place error logic for the year input (check if within bounds) inside the logic section of SalaryForm?

Reply:
Yes! :D 

So with any error states I'd pass that as a boolean or message prop. Sometimes you'd want custom messages, other times static/default. Given that this has a max/min value as well, we would give it an error type and have the message static inside the inputfield, we could also check is min+max is set then message should be x, if not message prop, or default y. 

If that made any sense 😅